ccd-definition/CaseEvent.json

```diff
@@ -7,7 +7,21 @@
     "Description": "Create a new case – add a title",
     "DisplayOrder": 1,
     "PostConditionState": "Open",
-    "SecurityClassification": "Public"
+    "SecurityClassification": "Public",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/case-initiation"
```

Comment:
TBH I though more of `AboutToStart` callback which is to be used to prepare data.

Only question is whether failure in `AboutToStart` would stop case from being able to be created.

Reply:
Submitted events are intended to show a case has transitioned into a new state. i.e non existant to created... I think it's the right type of callback for this sitution.

About to start callback is more for data validation from what I can see. Yes it would stop the event from happening if local authority was not set correctly.

Reply:
A `AboutToSubmit` event is dedicated for validation.

The way I look at this is if we had no CCD and some sort of fire and forget processing, regardless how robust that is, extracting LA name from user token would happen within main transaction, not as post processing. Similarly here I find `AboutToStart` more appropriate for what we doing.

Happy to discuss when I get back from the meeting.

Reply:
AboutToSubmit could be more appropriate. Reading docs we can put it into a different state if that local authority isn't set. We could maybe configure a manual select local authority field if this happens?